app/views/assessments/new.html.erb

```diff
@@ -65,17 +65,17 @@
             </div>
           </div>
 
-          <div class="field">
-            <span>Assessment Type</span>
+          <fieldset class="field-section">
+            <legend class="field-section__title">Assessment Category</legend>
             <div class="radio">
               <%= form.radio_button :category, 'phone triage', {:checked => @need[:category] == 'Phone triage'} %>
               <%= form.label :category_phone_triage, "Phone Triage", class: "radio__label" %>
             </div>
             <div class="radio">
-              <%= form.radio_button :category, 'check in', {:checked => @need[:category] == 'Check In'} %>
+              <%= form.radio_button :category, 'check in', {:checked => @need[:category] == 'Check in'} %>
```

Comment:
Small issue that would prevent your choice from being kept when the form was re-rendered after validation errors

Reply:
ah! that happened to me but I couldn't reproduce so I assumed I was going mad